Ontologies.Mappings/test/MappedMappingValidationTests.cs

```diff
@@ -17,11 +18,14 @@ public MappedMappingValidationTests(ITestOutputHelper output)
         }
 
         [Theory]
-        [InlineData("Mappings.v1.Mapped.willow_v1_dtdlv2_mapped.json")]
-        public void ValidateMappedDtmisAreValidFormat(string resourcePath)
+        [InlineData("https://mapped.com/ontologies/mapping/Willow2Mapped/latest.json", LoaderType.Http)]
```

Comment:
By using the FileLoader we can do all the testing locally, and then jsut have these tests be a regression test on what's actually published. But yeah, if you want to publish to a staging location during the PR build, that would be fine too. The good news is that these files don't change that often.

Reply:
From my understanding, the new tests appear to be referring to the mappings in blob storage rather than `Ontologies.Mappings/src/Mappings/v1`. Are you suggesting that we switch back to the file loader, or am I missing something?